WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -119,10 +119,10 @@ extension XCUIElement {
         return XCUIApplication().staticTexts.containing(predicate).count
     }
 
-    public func assertTextVisibilityCount(textToFind: String, expectedCount: Int) {
-        let count = try! getStaticTextVisibilityCount(textToFind: textToFind)
-        XCTAssertTrue(count == expectedCount, "Expected '\(textToFind)' to appear \(expectedCount) times, but it appeared '\(count)' times!")
-    }
+    public func assertTextVisibilityCount(textToFind: String, expectedCount: Int = 1) {
```

Comment:
I agree that adding the default value doesn't improve the explicitness/readability, however I needed a way to use `expectedCount: 2` since I'm calling this in SingleOrderScreen in order to check the first name, which appears twice in this case. Since most of the rest of the calls to this function do expect only 1 count, I made that the default.
Then, in SingleOrderScreen, I could assert:
`app.assertTextVisibilityCount(textToFind: order.billing.first_name, expectedCount: 2)`